cms/djangoapps/models/settings/course_metadata.py

```diff
@@ -272,6 +278,95 @@ def update_from_dict(cls, key_values, descriptor, user, save=True):
 
         return cls.fetch(descriptor)
 
+    @classmethod
+    def validate_team_settings(cls, settings_dict):
+        """
+        Validates team settings
+
+        :param settings_dict: json dict containing all advanced settings
+        :return: a list of error objects
+        """
+        errors = []
+        teams_configuration_model = settings_dict.get('teams_configuration', {})
+        if teams_configuration_model == {}:
+            return errors
+        json_value = teams_configuration_model.get('value')
+        if json_value == '':
+            return errors
+
+        proposed_max_team_size = json_value.get('max_team_size')
+
+        if proposed_max_team_size != '' and proposed_max_team_size <= 0:
```

Comment:
If there is no max_team_size (`proposed_max_team_size = None`) , this raises an exception

Reply:
fixed